src/app/ApplicationTemplate.UWP/App.xaml.cs

```diff
@@ -20,6 +21,10 @@ public App()
 			Startup = new Startup();
 			Startup.PreInitialize();
```

Comment:
It seems like Startup.PreInitialize() might be a good place to put the added code

Reply:
Startup seems to content only app configuration and no UNO related stuff.
But Because it's a flag configuration should we not keep them in the app.xaml.cs? 
Anyway yeah I placed it a little bit to late too :D

Reply:
Uno or not, I think all early configuration should go there

Reply:
alright moved !